correios/models/user.py

```diff
@@ -130,8 +130,8 @@ def __init__(self,
                  display_name: Optional[str] = "",
                  symbol: Optional[str] = None,
                  max_weight: Optional[int] = None,
-                 min_declared_value: Optional[Decimal] = Decimal("0.00"),
```

Comment:
`Optional` was removed because it means "this argument accepts `None` value" and not "this argument can be left empty and filled by default values". The usecase here was the latest, hence the change.